common/scripts/configs/master_integrations.sql

```diff
@@ -436,6 +436,12 @@ do $$
       values ('59e71aa80552d20500e76ba8', 82, 'ddcKey', 'Docker DataCenter', 'generic', false, 'account', 5012, '54188262bc4d591ba438d62a', '54188262bc4d591ba438d62a', '2017-10-31', '2017-10-31');
     end if;
 
+    -- webhookV2
+    if not exists (select 1 from "masterIntegrations" where "name" = 'webhookV2' and "typeCode" = 5012) then
+      insert into "masterIntegrations" ("id", "masterIntegrationId", "name", "displayName", "type", "isEnabled", "level", "typeCode", "createdBy", "updatedBy", "createdAt", "updatedAt")
+      values ('59e71aa80552d20500e76ba9', 83, 'webhookV2', 'Event Trigger', 'generic', false, 'account', 5012, '54188262bc4d591ba438d62a', '54188262bc4d591ba438d62a', '2017-11-03', '2017-11-03');
```

Comment:
This should be called `Webhook`, not `Event Trigger`. Please look at what's in the heap.